src/tracker.jl

```diff
@@ -0,0 +1,203 @@
+module TrackerAD
```

Comment:
maybe move this file to `compat/tracker.jl` and load it optimally with `Requires.jl`.